README.md

```diff
@@ -1,135 +1,170 @@
-# Diarizers
+# 🤗 Diarizers
 
-Diarizers is a library for fine-tuning [`pyannote`](https://github.com/pyannote/pyannote-audio/tree/develop) speaker diarisation models using Hugging Face tools. It provides scripts and tools to: 
-
-- Convert Speaker Diarization datasets into Hugging Face datasets compatible with the `diarizers` library. 
-- Fine-tune speaker diarization models using the Hugging Face `Trainer`. 
-- Test the models at inference time. 
-- Convert the fine-tuned models into a format compatible with `pyannote`.  
-
-> [!IMPORTANT]
-> For now, this library can only be used to fine-tune the [segmentation model](https://huggingface.co/pyannote/segmentation-3.0) from the [speaker diarization pipeline](https://huggingface.co/pyannote/speaker-diarization-3.1). 
-> Future work will aim to help optimise the hyperparameters of the entire pipeline. 
+🤗 Diarizers is a library for fine-tuning [`pyannote`](https://github.com/pyannote/pyannote-audio/tree/main) speaker 
+diarization models using the Hugging Face ecosystem. It can be used to improve performance on both English and multilingual 
+diarization datasets with simple example scripts, with as little as ten hours of labelled diarization data and just 5 minutes
+of GPU compute time.
 
 ## 📖 Quick Index
 * [Installation](#installation)
 * [Train](#train)
-* [Inference](#inference)
+* [Evaluation](#evaluation)
+* [Inference](#inference-with-pyannote)
 * [Results](#Results)
-* [Adding new datasets](#addingnewdatasets)
+* [Adding new datasets](#adding-new-datasets)
 * [Acknowledgements](#acknowledgements)
 * [Citation](#citation)
 
 ## Installation
 
-Diarizers has light dependencies and can be installed with the following lines: 
+First, clone the repository and install the dependencies:
 
 ```sh
-pip install git+https://github.com/huggingface/diarizers.git
-pip install diarizers[dev]
+git clone https://github.com/huggingface/diarizers.git
+cd diarizers
+pip install -e .
 ```
 
-You'll need to generate a [user access token](https://huggingface.co/docs/hub/en/security-tokens), and login with: 
+To load pre-trained diarization models from the Hub, you'll first need to accept the terms-of-use for the following two models:
+1. [pyannote/segmentation-3.0](https://hf.co/pyannote/segmentation-3.0)
+2. [pyannote/speaker-diarization-3.1](https://hf.co/pyannote/speaker-diarization-3.1)
+
+And subsequently use a Hugging Face authentication token to log in with: 
 
 ```
-pip install -U "huggingface_hub[cli]"
-hugging-cli login
+huggingface-cli login
 ```
 
 ## Train
 
-When fine-tuning a `pyannote` segmentation model on a given dataset, be sure to specify this: 
```

Comment:
We need to give more context here over what the script is actually doing, before introducing these arguments. Otherwise the reader has no idea what these arguments are referring to